lumispy/tests/signals/test_luminescence_spectrum.py

```diff
@@ -68,6 +68,10 @@ def test_errors_raise(self):
     def test_warnings(self):
         pytest.importorskip("hyperspy_gui_ipywidgets")
         s = LumiSpectrum(np.ones(50))
-        with pytest.warns(SyntaxWarning) as warninfo:
+        with pytest.warns(SyntaxWarning, match='Using the Hyperspy') as warninfo:
```

Comment:
no need for the `as warninfo` (here and below) then, because we do not need to check that variable